test-d/value-of.ts

```diff
@@ -0,0 +1,7 @@
+import {expectType, expectError} from 'tsd';
+import {ValueOf} from '..';
+
+const value: ValueOf<[1, 2, 3]> = 1;
```

Comment:
```suggestion
const value: ValueOf<{ key: 3 }> = 3;
```

Why not do the same here for consistency?

Reply:
Done!